media-sound/musescore/musescore-3.0.0_git.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="An open source and free music notation software"
+DESCRIPTION="This aplication is used for creating, playing and printing beautiful sheet music"
+HOMEPAGE="https://musescore.org"
+COPYRIGHT="Free Software Foundation"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/musescore/MuseScore/archive/8d8ceee1bd208cb2478b91d13ef843bc0980be7b.zip"
+CHECKSUM_SHA256="0acf4de2752603595a0943e1d5fa267565df3c7c536a1d4637db6bccd3c0faf5"
+SOURCE_DIR="MuseScore-8d8ceee1bd208cb2478b91d13ef843bc0980be7b"
+PATCHES="musescore-$portVersion.patchset"
+ARCHITECTURES="!x86_gcc2 x86"
```

Comment:
There should be an empty line above ARCHITECTURES.
While at it, could you please change it to:
```
ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
```